assets/js/admin/advertise-asc-components/campaign-edit-view.js

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from 'react';
+import { useState } from '@wordpress/element';
 import { Card, Col, Form, Input, InputNumber, Row, Select, Space, Switch, Tooltip } from 'antd';
```

Comment:
❓ Any reason to use ant design components instead of wordpress/element ?

Reply:
Just familiarity with the ant framework and the tight deadline.

Reply:
I understand, but Facebook for WooCommerce is one of our canonical extensions and we'd prefer if we make use of woocommerce/components? I am trying to avoid dependency on a third-party library we might have to maintain in the long run.

Could you have a look at https://www.npmjs.com/package/@woocommerce/components?activeTab=readme maybe and let me know if this is something you'd consider?

Reply:
We could consider it as a tech debt and switch when there's time, but for the current PR I don't think I can do it.

Reply:
Seems like the woocommerce/components package requires a higher version of php, etc. than what we have atm. 
We are on php 7.1 and also WP 14. So the plugin needs some housekeeping before we can use that @woocommerce/components package.

Reply:
> We are on php 7.1

No we are on 7.4:

https://github.com/woocommerce/facebook-for-woocommerce/blob/51cbb2fd7554f168b2410494d1ce95f782ae28c2/composer.json#L13

What is WP 14?

Reply:
Seems like we are on two different branches. Or maybe mine is a bit too old

Reply:
a simple installation of the woocommerce/components package shows many conflicts with the current @wordpress package that's being used. I cannot simply test everything before making such a change. This can risk the timely release of ASC

Reply:
I think this wasn't mentioned yet - there is also [`@wordpress/components`](https://www.npmjs.com/package/@wordpress/components) package (see [storybook](https://wordpress.github.io/gutenberg/)).

Using `@wordpress/components` - combined with https://www.npmjs.com/package/@wordpress/dependency-extraction-webpack-plugin that should already be setup in your use of https://www.npmjs.com/package/@wordpress/scripts - should make the plugin significantly lighter in size.

Reply:
Nice. I'll take a look.